keycloak.env

```diff
@@ -1,3 +1,7 @@
-KEYCLOAK_CLIENT_ID=itvr-2674
-KEYCLOAK_REALM=onestopauth-basic
-KEYCLOAK_URL=https://dev.oidc.gov.bc.ca/auth/
+BCEID_KEYCLOAK_CLIENT_ID=itvr-2674
+BCEID_KEYCLOAK_REALM=onestopauth-basic
+BCEID_KEYCLOAK_URL=https://dev.oidc.gov.bc.ca/auth/
+
+BCSC_KEYCLOAK_CLIENT_ID=zeva
```

Comment:
I just realised - we should create a new client here for itvr and not have bcsc auth in this zeva client.